config/credentials.yaml

```diff
@@ -17,3 +17,9 @@ credentials:
         domain:
           description: "GitHub API domain"
           name: "api.github.com"
+      - credentials:
+          - string:
+              scope: GLOBAL
+              id: slack-token
+              secret: '${SLACK_TOKEN}'
+              description: Slack token
```

Comment:
Can you align the order of fields and the double quote on the format of the existing credentials?